files/disable_inactive_freeipa_users.sh

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/bash
+
+# Usage:
+#   disable_inactive_freeipa_users.sh [disable_time]
+#
+# Users are considered inactive if they have not logged in since
+# disable_time.  If disable_time is not specified then a default value
+# of "45 days ago" is used.
+#
+# The date/time must be specified so that it is understandable by the
+# date command, which is the same as saying it must be understandable
+# by coreutils.  See the "Date input formats" section of the info
+# manual distributed with coreutils for more information on date/time
+# formats:
+# https://www.gnu.org/software/coreutils/manual/html_node/Date-input-formats.html
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# Users are considered inactive if they have not logged in since this
+# time.
+#
+# See the "Shell Parameter Expansion" section of the info manual
+# distributed with bash for more details on this form of parameter
+# assignment:
+# https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html
+disable_time=${1:-45 days ago}
+
+# Use a temp file for our Kerberos credentials cache, since it only
+# needs to exist while this script is running.
+KRB5CCNAME=$(mktemp)
+
+function usage {
+  cat << HELP
+Usage:
+  ${0##*/} [disable_time]
+
+Users are considered inactive if they have not logged in since
+disable_time.  If disable_time is not specified then a default value
+of "45 days ago" is used.
+
+The date/time must be specified so that it is understandable by the date
+command, which is the same as saying it must be understandable by
+coreutils.  See the "Date input formats" section of the info manual
+distributed with coreutils for more information on date/time formats:
+https://www.gnu.org/software/coreutils/manual/html_node/Date-input-formats.html
+HELP
+  exit 1
+}
+
+if [ $# -gt 1 ]; then
+  usage
+else
+  # Convert disable_time into an integer representing seconds since
+  # the epoch.
+  disable_deadline=$(date --date="$disable_time" +%s)
+
+  # kinit via the host's keytab.
+  kinit -k -t /etc/krb5.keytab
+
+  # Grab a list of all non-disabled FreeIPA users
+  users=$(
+    # Note that we disable the size and time limits normally in place
+    # when running ipa user-find.
+    ipa user-find --disabled=false --sizelimit=0 --timelimit=0 \
+      |
+      # --quiet means no printing unless the p command is used
+      sed --quiet 's/^\s*User login:\s*//p'
```

Comment:
Same question as the previous.

```suggestion
      # --quiet means no printing unless the p command is used
      | sed --quiet 's/^\s*User login:\s*//p'
```